src/ui/LeftSearchPanel.java

```diff
@@ -0,0 +1,110 @@
+package ui;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.File;
+
+
+/**
+ * This is the panel that all the images will be loaded in a small icons list
+ */
+public class LeftSearchPanel extends JPanel {
+
+    private JToolBar jToolBar;
+    private ImageIcon iconPath;
+    private JButton chooseFileButton;
+    private File selectedFile;
+    private JList imagesList;
+    private ImageListModel imageListModel;
+    private JScrollPane scrollPaneJlist;
+    private RightPanelViewer rightPanelViewer;
+
+    /**
+     * constructor load all static metods and apply  a Border layout
+     *
+     * @param rightPanelViewer this object is the rigth panel, We instantiate it to use some methods of the panel
+     */
+    public LeftSearchPanel(RightPanelViewer rightPanelViewer) {
+        this.rightPanelViewer = rightPanelViewer;
+        this.setLayout(new BorderLayout());
+        toolbar();
+    }
+
+    /**
+     * this method is the toolbar which will be able to choose the folders that images are loaded
+     */
+    public void toolbar() {
+        iconPath = new ImageIcon(ToolBar.class.getResource("../img/folderOpen.gif"));
+        chooseFileButton = new JButton(iconPath);
+        chooseFileButton.addActionListener(new ButtonFile());
+        jToolBar = new JToolBar();
+        jToolBar.add(chooseFileButton);
+        this.add(jToolBar, BorderLayout.NORTH);
+    }
+
+    /**
+     * this methods instance the Jlist and contains the event mouse clicked who is in charge to select the image that will be displayed
```

Comment:
the comments should not be bigger than 100 characters, split this in different lines